pandas/tests/frame/test_indexing.py

```diff
@@ -3010,6 +3010,14 @@ def test_mask_callable(self):
         tm.assert_frame_equal(result,
                               (df + 2).mask((df + 2) > 8, (df + 2) + 10))
 
+    def test_mask_list(self):
```

Comment:
can you pas a bunch of list-likes in a fixture (the old box argument)

Reply:
Old box argument?  Sorry, I'm not sure what you mean here (or I forgot what it meant if you have mentioned it to me before).